docs/administration/garbage-collection/_index.md

```diff
@@ -37,7 +38,8 @@ You are able to schedule garbage colleciton on your Harbor interface from the **
     * **Weekly**: Run garbage collection at midnight every Saturday.
     * **Custom**: Run garbage collection according to a `cron` job.
 
-1. To **Allow garbage collection on untagged artifacts**, select the option. If you have this option turned on, the next time Garbage collect runs on your Harbor instance, Harbor will delete untagged artifacts and then perform garbage collection on them.     
+1. To **Workers**, select the option. Set the number of workers that can execute GC tasks in parallel, the default value is 1.
+1. To **Allow garbage collection on untagged artifacts**, select the option. If you have this option turned on, the next time Garbage collect runs on your Harbor instance, Harbor will delete untagged artifacts and then perform garbage collection on them.
```

Comment:
Can you also specify here what I tagged artifacts are. Specifically regarding latest tag?

Reply:
No changes have been made to `Allow garbage collection on untagged artifacts`. `Allow garbage collection on untagged artifacts` will not affect tagged artifacts or the latest tag. These artifacts will not be GC.

Reply:
The question was to improve the documentation in general. Because it is not clear to the user what **untagged artifacts,** means. https://github.com/goharbor/harbor/issues/16326

Reply:
This PR only add GC worker related descriptions, can you open a new issue or PR to track it down for other issues?